app/graphql/schema/schema.ts

```diff
@@ -1,8 +1,8 @@
-import { makeAugmentedSchema } from "@neo4j/graphql"
+import { Neo4jGraphQL } from "@neo4j/graphql"
```

Comment:
Alpha introduced a breaking change from previous versions - and this is the most important one to get in place 🤓

Feel free to look at the "Breaking changes from alpha.4" section in https://www.notion.so/neo4j-graphql-v1-0-0-alpha-5-a87c7963ff9f4773ba37b9768f1c1cba